src/components/va-accordion/va-accordion-item.e2e.ts

```diff
@@ -0,0 +1,46 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('va-accordion-item', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-accordion-item></va-accordion-item>');
+    const element = await page.find('va-accordion-item');
+
+    expect(element).toEqualHtml(`
+    <va-accordion-item class="hydrated">
+      <mock:shadow-root>
+        <button aria-controls="content" aria-expanded="false"></button>
+        <div id="content">
+          <slot></slot>
+        </div>
+      </mock:shadow-root>
+    </va-accordion-item>`);
+  });
+
+  it('sets aria-expanded to true based on prop', async () => {
```

Comment:
Should we also test that the item contents are expanded, or is that...just kinda always on (hidden only by CSS)?

I don't suppose you know of a way to test the display of the content, eh? Like, the `display` CSS property. Because while this tests the `aria-expanded` attribute, it makes the assumption that the styling for that attribute _does_ something, y'know? That's probably way out of scope for this PR, though. :man_shrugging:

Reply:
I would rather test that using screenshot diffs than test for the presence of CSS properties. That seems....awkward.